src/app/components/named-entity/named-entity.component.html

```diff
@@ -16,35 +16,34 @@
         {{ entity.label }}
         <evt-pinner [item]="entity" renderer="NamedEntity" pinType="NamedEntities"></evt-pinner>
     </div>
-    <div class="card-body ne-content" *ngIf="contentOpened">
-        <!-- Tab panes -->
-        <ngb-tabset [justify]="false">
-            <ngb-tab [title]="'Info' | translate">
-                <ng-template ngbTabContent>
+    <div class="card-body ne-content d-none" [class.d-block]="contentOpened">
+        <ngb-tabset [justify]="false" #entityDetails="ngbTabset">
+            <ngb-tab id="info" [title]="'Info' | translate" [disabled]="entity.content.length === 0">
+                <ng-template ngbTabContent *ngIf="contentOpened">
                     <div class="ne-tab-content">
                         <evt-content-viewer *ngFor="let detail of entity.content" [content]="detail">
                         </evt-content-viewer>
                     </div>
                 </ng-template>
             </ngb-tab>
-            <ngb-tab [title]="'Occurrences' | translate">
-                <ng-template ngbTabContent>
+            <ngb-tab id="occurrences" [title]="'Occurrences' | translate">
```

Comment:
How does this `id` behave when multiple instances of this component are presenti in the page?

Reply:
In Bootrap documentation they say that `id` is «The tab identifier. Must be unique for the entire document for proper accessibility support». But it works correctly when multiple instances of this component are present in the page, even if I use the same id for tabs. Shall I use the entity id to create a more appropriate, unique identifier?

Reply:
It is probably better to not rely on implementation behaviour. Let's put unique ids ;)

Reply:
Should I generate them programmatically or do you think I can use NamedEntity ID? It should be unique!

Reply:
since you have the NE-id just use that